api_app/models.py

```diff
@@ -104,8 +103,6 @@ def init_report(cls, name, job_id):
             job=cls.object_by_job_id(job_id),
             report={},
             errors=[],
-            start_time=timezone.now(),
```

Comment:
uncomment this

Reply:
Shouldn't adding default value to `start_time` and `end_time` fields as `timezone.now` be better ?

Reply:
it makes sense to add for `start_time`